modules/heat_transfer/src/base/HeatConductionApp.C

```diff
@@ -0,0 +1,10 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "HeatTransferApp.h"
```

Comment:
Seems like this file is unnecessary. But if it's necessary
```suggestion
#include "HeatConductionApp.h"
```

Reply:
yeah that s what I was trying to do
I want that header to be included at least once so it s compiled